README.md

```diff
@@ -9,6 +9,12 @@ Kubernetes cluster.  The basic usage is:
 
 This chart is tested with k8s 1.21+ and helm 3.6.0+.
 
+# NOTICE: Future Deprecation
+
+We have plans in the very near future to deprecate this chart in favor of a [new git repo](https://github.com/mastodon/helm-charts), which has proper helm repository support (e.g. `helm repo add`), and will contain multiple charts, both for mastodon and for supplementary components that we make use of.
+
+We still encourage suggestions and PRs to help make this chart better, and this repository will remain available after the new charts are ready to give users time to migrate. However, we will not be approving large PRs, or PRs that change fundamental chart functions, as those changes should be directed to the new charts.
```

Comment:
is the new chart not going to start with an import of this one?

Reply:
also why not just rename this repo, and move the  existing chart into a `charts/` subdirectory?

Reply:
It's going to be a "rough" import, as we basically want to do a refactor of this chart to clean it up.

We can't rename the repo because we additionally want it this repo to both support multiple helm charts in the future, and support `helm repo` integration, neither of which seem possible with an existing repo.

But this one will be around for a while afterwards. I just made a pinned issue explaining this in more detail: #129